### PR TITLE
Created a unit test for polygonal-face-tessellation

### LINKED ISSUE #61: Example file "polygonal-face-tessellation" not rendered.
The file is [here](../blob/development/testdata/IFC4x1ExampleFiles/official/E4_Tessellated_shape/polygonal-face-tessellation.ifc).  OIP doesn't show anything.

### LINKED ISSUE #317: [BUG] Official IfcPolygonalFaceSet example is wrong
**Describe the bug**
See [here](https://forums.buildingsmart.org/t/ifcpolygonalfaceset-and-face-winding/3169/3)

**Expected behavior**
- [ ] Correct the example file, so that face winding is correct.
- [ ] Update unit tests if needed.

### PR DESCRIPTION
**Pull request by SamuilsRulovs**
Fixes #61 closes #317

supersedes #342

### CODE REVIEW

File: UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	std::shared_ptr<oip::EXPRESSModel> express_model = nullptr;
+	buw::ReferenceCounted<oip::IfcImporterT<emt::IFC4X1EntityTypes>> importer = nullptr;
+};
+
+TEST_F(PolygonalFaceTessellation, AllEntitiesAreRead) {
+	EXPECT_THAT(express_model->entities.size(), Eq(43));
+}
+
+TEST_F(PolygonalFaceTessellation, IFCHasAnEssentialEntity) {
+	auto result = std::find_if(express_model->entities.begin(), express_model->entities.end(), [](auto &pair) -> bool { return pair.second->classname() == "IFCPOLYGONALFACESET"; });
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp]: Please also test for the existence of IfcIndexedPolygonalFace and ...WithVoids.

**Review by pjanck — CHANGES_REQUESTED** [review]
Please also correct the corresponding readme that we now support these entities.

File: UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp (hunk 1 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp]: Resolved:  2af454e

**Review by jschlenger — COMMENTED** [review]
Looks good.

Did you add the checkmarks to the "Documentation/markdown/SupportedIFCrepresentations.md" file?

**Comment by SamuilsRulovs** [comment]
> Looks good.
> 
> Did you add the checkmarks to the "Documentation/markdown/SupportedIFCrepresentations.md" file?

Thank you for reminder! I have just done it.

**Review by jschlenger — APPROVED** [review]
Perfect!

**Review by pjanck — APPROVED** [review]
LFTM

**Pull request merged**